scheds/rust/scx_layered/src/main.rs

```diff
@@ -1246,6 +1268,211 @@ impl Layer {
         Ok(nr_new_cpus)
     }
 }
+#[derive(Debug, Clone)]
+struct NodeInfo {
+    node_mask: nix::sched::CpuSet,
+    _node_id: usize,
+}
+#[derive(Debug)]
+struct GpuTaskHandler {
```

Comment:
Overall comment explaining what the struct does would be great. Maybe the name is too generic. `GpuTaskAffinitizer`?